shared/javascripts/local_storage.js

```diff
@@ -87,14 +87,21 @@ var ls = {
   }
 };
 
-// Determine whether localstorage can be used directly
-try { 
-  localStorage;
-} catch(e) {
-  ls.localStorageDefined = false;
-  
-  // Assuming Xul firefox
-  ls.preferences = Components.classes["@mozilla.org/preferences-service;1"]
-                         .getService(Components.interfaces.nsIPrefService)
-                         .getBranch("extensions.privly.");
+var isCommonJS = typeof window == "undefined" && typeof exports == "object";
```

Comment:
You should use strict equality checks `===` in almost every case with JavaScript.